CefSharp/Handler/IPermissionHandler.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2022 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+
+namespace CefSharp
+{
+    /// <summary>
+    /// Handle events related to permission requests. 
```

Comment:
Update to match CEF comment https://github.com/chromiumembedded/cef/blob/master/include/cef_permission_handler.h#L83

Will also need added disclaimer about CEF  UI thread by default not being the same as your application UI Thread. (Copy and paste that from one of the others that has it).